vllm/engine/llm_engine.py

```diff
@@ -229,6 +250,7 @@ def _init_workers_ray(self, placement_group: "PlacementGroup",
 
         distributed_init_method = get_distributed_init_method(
             driver_ip, get_open_port())
+        mscclpp_init_method = f"eth0:{driver_ip}:{get_open_port()}" if self.parallel_config.sep_prompt_token else None
```

Comment:
minor: can we leave a TODO here. 
1. I feel this is limited if we want to use high speed network interface. It would be great to extract and env like `NCCL_SOCKET_IFNAME`
1. Do not know whether mscclpp is compatible with other high speed interfaces?

Reply:
Besides, I don't think everyone's default network interface would have eth0. Agree with @Jeffwan's suggestion. Or at least let user to specify which network interface they would like to use.